flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
PyTorch is used as dependency which is packaged for either x86_64 or aarch64. Also, compiling it from source is quite complicated, since CUDA and cuDNN is needed, and CUDA is embedded into the package during compilation.
Even though CUDA is currently not supported in the application, it might be in the following releases when the Flatpak sandboxing issue gets fixed and I get access to NVIDIA hardware again.

Reply:
since it exist for aarch64 whu restricting to just x86_64....

Reply:
as per the doc, `only-arches` can be used for `sources` entries and `module` entries.